BraveShared/Analytics/DAU.swift

```diff
@@ -142,7 +152,13 @@ public class DAU {
         
         let lastPingTimestamp = [Int((today).timeIntervalSince1970)]
         
-        return ParamsAndPrefs(queryParams: params, lastLaunchInfoPreference: lastPingTimestamp)
+        var headers: [String: String] = [:]
+        
+        if let key = self.apiKey, !key.isEmpty {
+            headers["x-brave-api-key"] = key
```

Comment:
this should be added to https://github.com/brave/brave-browser/wiki/Custom-Headers

Reply:
@diracdeltas As far as I know this isn't for any websites, just for the DAU ping, so not sure if we want to add it to that custom headers wiki